### PR TITLE
Lint for `.only` tests

### PR DESCRIPTION
**Pull request by stefvhuynh**
https://github.com/FormidableLabs/victory/issues/174

### CODE REVIEW

**Comment by boygirl** [comment]
@stefvhuynh looks good to me! Approved

File: package.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
     "lint-source": "eslint --color --ext .js,.jsx src",
     "lint-demo": "eslint --color --ext .js,.jsx demo",
     "lint-docs": "eslint --color --ext .js,.jsx docs",
-    "lint-test": "eslint --color --ext .js,.jsx test",
+    "lint-test": "eslint --color --ext .js,.jsx --rule 'mocha/no-exclusive-tests:2' test",
```
Comment (ryan-roemer) [inline package.json]: We shouldn't have a command line rule here. it should be in `config/eslint/.eslintrc-test`

**But**, it appears on all these lints we're _not_ using our configs???

Comment (boygirl) [inline package.json]: The `.eslintrc` in each project points to the archetype configs. Is that incorrect?

Comment (boygirl) [inline package.json]: I think we went that route so that we would have an `.eslintrc` in each project for in-editor linters to use

**Comment by boygirl** [comment]
@stefvhuynh my mistake. Please move the rule into the config. You can reference the pattern shown here https://github.com/FormidableLabs/builder-victory-component/blob/master/config/eslint/.eslintrc-docs

File: config/eslint/.eslintrc-test [hunk 2]
```diff
@@ -6,6 +6,12 @@ extends:
 env:
   mocha: true
 
+plugins:
+  - mocha
+
+rules:
+  mocha/no-exclusive-tests: 2
```
Comment (boygirl) [inline config/eslint/.eslintrc-test:13]: To match the rest, let's have this as 

```
rules:
  mocha/no-exclusive-tests:
    - 2
```

**Comment by boygirl** [comment]
Approved pending nitpicky change

File: config/eslint/.eslintrc-test (hunk 2 above)
Comment (ryan-roemer) [inline config/eslint/.eslintrc-test:13]: Actually, it should be left as is. It is a scaler value it is not an array like the other things in this file

**Pull request merged**